### PR TITLE
Small terminology update

### PR DESCRIPTION
**Pull request by twasyl**
Replacing a `master` by `controller`.

<a href="https://gitpod.io/#https://github.com/jenkinsci/jenkins/pull/7271"></a>

### CODE REVIEW

**Review by NotMyFault — APPROVED** [review]

**Review by res0nance — APPROVED** [review]

**Comment by NotMyFault** [comment]
This PR is now ready for merge. We will merge it after ~24 hours if there is no negative feedback.
Please see the [merge process documentation](https://github.com/jenkinsci/jenkins/blob/master/docs/MAINTAINERS.adoc#merge-process) for more information about the merge process.
Thanks!

**Review by Wadeck — APPROVED** [review]
👍 good catch

**Pull request merged**
